server/routes/induction/dynamicAriaTextResolver.ts

```diff
@@ -18,6 +18,7 @@ const getDynamicBackLinkAriaText = (req: Request, backLinkUrl: string): string =
     '/plan/{PRISON_NUMBER}/view/education-and-training': `Back to ${prisonerName}'s learning and work progress`,
     '/plan/{PRISON_NUMBER}/view/work-and-interests': `Back to ${prisonerName}'s learning and work progress`,
     '/prisoners/{PRISON_NUMBER}/induction/hoping-to-work-on-release': `Back to Is ${prisonerName} hoping to get work when they're released?`,
+    '/prisoners/{PRISON_NUMBER}/induction/previous-work-experience': `Back to What type of work has ${prisonerName} done before?`,
```

Comment:
This file (`dynamicAriaTextResolver`) is used to produce the aria labels for the Back links.
Basically its a lookup of URLs to aria label.

(don't shoot the messenger about whether the aria labels actually make sense / read sensibly !! the scope of our work here is to migrate the Induction journey functionally as-is from the CIAG UI. These are the aria labels that the CIAG UI currently implements, and presumably were tested and signed off by someone? 🤷‍♂️ . I'm all up for changing these labels, but not in the scope of this "as-is" migration)

Reply:
Yeah, agreed. The whole Back link thing is a bit of a mess IMHO.
In the scope of this work we are to migrate as-is, warts and all! If we make functional changes, even cosmetic/wording changes, it will be really hard during testing to determine if we have regressed something, or if the reported problem was always part of the CIAG UI

Personally I have a massive problem with Back links like this. I know they are part of GDS and all that, but technically I think they are flawed and can make the problem they try to solve worse!
From a technical point of view we need to track the user's history and previous page ... which depending on business requirements may not always be the browser page the user last saw (difficult to explain, but we have a couple of examples of it in these Induction flows, hence us having to track the state with the `pageFlowHistory` object in the session.
The 2nd problem I have is that from a browser history point of view, using these Back links does not actually move you "back" at all! They are regular anchor links, so actually move you forwards and add to the browser history!
This then becomes massively confusing for the user when they use a combination of the application Back link, and then the browser Back and Forward buttons (or worse, gestures on their laptop trackpad or magic mouse!)
It's a nightmare! 😱  

I think once we have migrated these screens (warts and all), we'll have an opportunity to re-address some of this stuff. I know there is some work coming our way to change the question set. This will involve simplifying the flow so there is no longer a "long" and "short" question set, change the order of some questions, and remove some others.
I think at that point we'll have our opportunity to ask questions about the Back links etc including text labels etc 👍